repository/src/org/pentaho/platform/security/userroledao/jackrabbit/AbstractJcrBackedUserRoleDao.java

```diff
@@ -886,4 +893,17 @@ private boolean canDeleteRole( Session session, final IPentahoRole role ) {
     }
     return usersToBeRemoved.toArray( new String[0] );
   }
+
+  public boolean getUseJackrabbitUserCache() {
```

Comment:
I had used "get" instead of "is" because in earlier Spring versions had troubles with identifying of beens wich use getters started with "is". In the current version it's not the issue. PR is updated. Thanks.